libs/platform/src/lib/components/dynamic-page/dynamic-page.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs';
+
+@Injectable()
+export class DynamicPageService {
+    private toggle = new Subject<boolean>();
+    private expand = new Subject<void>();
+    private collapse = new Subject<void>();
+
+    public $toggle = this.toggle.asObservable();
+    public $expand = this.expand.asObservable();
+    public $collapse = this.collapse.asObservable();
+
+    public toggleHeader(val: boolean): any {
```

Comment:
For what we have the `val` parameter here?
`Toggle` just should toggle to an opposite state.

Reply:
`toggleHeader` does not take value now, simply toggles.